ui/desktop/src/utils/pricing.ts

```diff
@@ -0,0 +1,60 @@
+import { getApiUrl } from '../config';
+
+export interface ModelCostInfo {
```

Comment:
should we make this a type in the openapi api and get it from there instead? that way we could use the generated SDK for fetching it too.